beacon/goclient/goclient.go

```diff
@@ -53,6 +53,7 @@ func New(opt beacon.Options) (beacon.Beacon, error) {
 		network:  core.NetworkFromString(opt.Network),
 		client:   autoClient,
 		graffiti: []byte("BloxStaking"),
+		blockChannel: make(chan spec.SignedBeaconBlock),
```

Comment:
@nivBlox why did we add it? what was a before that?

Reply:
yes it was but for now we disabled it as discussed

Reply:
@nivBlox so should we remove it?

Reply:
removed